lib/client.ml

```diff
@@ -43,10 +52,11 @@ let serve client msg_handler =
   Lwt.async server
 
 let init ~state ~msg_handler (address, port) =
-  let%lwt socket = Net.create_socket port in
+  let open Util.Net in
```

Comment:
See previous comment about module opening, but know that it's not a must for me.